tools/depends/target/exiv2/0001-WIN-lib-postfix.patch

```diff
@@ -1,6 +1,6 @@
 --- a/src/CMakeLists.txt
 +++ b/src/CMakeLists.txt
-@@ -215,6 +215,7 @@
+@@ -213,6 +213,7 @@
```

Comment:
I think they merged this but I probably forgot to backport

Reply:
Its not in the 0.28.x branch, only in master/nightly branches it seems.